challenger.go

```diff
@@ -41,8 +41,8 @@ type InvoiceClient interface {
 // LndChallenger is a challenger that uses an lnd backend to create new LSAT
 // payment challenges.
 type LndChallenger struct {
-	client        InvoiceClient
-	genInvoiceReq InvoiceRequestGenerator
+	Client                  InvoiceClient
```

Comment:
nit: exported fields should have godoc comments. This also removes the need for the large indentation as the fields will be separated a bit.